modules/dailymotionBidAdapter.js

```diff
@@ -38,17 +45,22 @@ function getVideoMetadata(bidRequest, bidderRequest) {
   const videoMetadata = {
     description: videoParams.description || '',
     duration: videoParams.duration || 0,
-    iabcat1: Object.keys(iabcat1),
+    iabcat1: Array.isArray(videoParams.iabcat1)
```

Comment:
ortb content has `cat` see section 5.1 
Per spec:
cat - string array - Array of IAB content categories that describe the content
producer. Refer to List 5.1.

Reply:
Added fallback to `cat`